frontend/packages/console-shared/src/components/getting-started/GettingStartedCard.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Flex,
+  FlexItem,
+  Text,
+  TextVariants,
+  Button,
+  SimpleList,
+  Skeleton,
+} from '@patternfly/react-core';
+import { ArrowRightIcon, ExternalLinkAltIcon } from '@patternfly/react-icons';
+import simpleListStyles from '@patternfly/react-styles/css/components/SimpleList/simple-list';
+
+import { GettingStartedLink } from './types';
+
+import './GettingStartedCard.scss';
+
+function ListLink({ loading, title, external, href, onClick }: GettingStartedLink) {
+  if (loading) {
+    return <Skeleton fontSize="sm" />;
+  }
+  if (onClick) {
+    return (
+      <button type="button" onClick={onClick} className={simpleListStyles.simpleListItemLink}>
+        {title}
+        <ArrowRightIcon aria-hidden="true" />
+      </button>
+    );
+  }
+  if (external) {
+    return (
+      <a
+        href={href}
+        target="_blank"
+        rel="noopener noreferrer"
+        className={simpleListStyles.simpleListItemLink}
+      >
+        {title}
+        <ExternalLinkAltIcon aria-hidden="true" />
+      </a>
+    );
+  }
+  return (
+    <Link to={href} className={simpleListStyles.simpleListItemLink}>
+      {title}
+      <ArrowRightIcon aria-hidden="true" />
+    </Link>
+  );
+}
+
+function MoreLink({ title, external, href, onClick }: GettingStartedLink) {
```

Comment:
Doesn't work here because I wanted to test this in the unit tests. But I removed this inline component now and just render the SimpleListItem in the loop of the GettingStartedCard.

See https://github.com/jerolimov/console/blob/odc-5680/frontend/packages/console-shared/src/components/getting-started/GettingStartedCard.tsx#L57-L81